clients/vscode/resources/config.schema.json

```diff
@@ -85,6 +92,31 @@
       },
       "required": []
     },
+    "Config__IndexConfig": {
+      "type": "object",
+      "properties": {
+        "dirs": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "excludeDirs": {
+          "anyOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            {
+              "type": "null"
```

Comment:
Is `null` the default / can you specify one of these objects without providing an `excludeDirs`?